querybook/server/lib/query_executor/base_executor.py

```diff
@@ -601,11 +601,11 @@ def _handle_exception(self, e, stack_trace: str):
             pass
         finally:
             # Update logger
-            error_type, error_str, error_extracted = self._parse_exception(e)
+            error_type, error_str, error_extracted = self._parse_exception(exc)
```

Comment:
from the [parse](https://github.com/pinterest/querybook/blob/214118c2137dfec580cba5dbeb4e0a19351f3c62/querybook/server/lib/query_executor/utils.py#L26) function, looks like this `error_type` is always `QueryExecutionErrorType.INTERNAL.value`, instead passing the `exc` to the format function, how about do some mappings for the error_type?

Reply:
resolved offline